.github/workflows/pr-build.yml

```diff
@@ -5,9 +5,16 @@ on:
   workflow_dispatch:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
-    paths-ignore:
-      - '**/*.md'
-      - README.md
+    paths:
+      - '**.ts'
+      - '.github/workflows/pr-build.yml'
+      - 'install/friendly-filenames.json'
+      - 'public/**'
+      - 'wing/**'
+      - Makefile
+      - tsconfig.json
+      - .npmrc
+      - .gitmodules
```

Comment:
Not exhaustively listing all possible scenarios, such as *.tsx, package.json, etc...